lib/jobs.rb

```diff
@@ -38,6 +48,14 @@ def self.query_eligible?(query)
     query =~ /\b#{JOB_RELATED_KEYWORDS}\b/i && !(query =~ /\b#{BLOCKED_KEYWORDS}\b/i) && !(query =~ /["():]|^-| -\S+/)
   end
 
+  def self.params(options)
+    { Keyword:        scrub_keyword(options[:query]),
+      Organization:   options[:organization_codes],
+      LocationName:   options[:location_name],
+      ResultsPerPage: options[:results_per_page]
+    }
```

Comment:
please re-fix the bracket alignment per https://github.com/GSA/search-gov/pull/187#discussion_r235543473

Reply:
weird thought I fixed that.